main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopTimeArrivalAndDepartureTimeValidator.java

```diff
@@ -53,6 +57,13 @@ public void validate(NoticeContainer noticeContainer) {
         GtfsStopTime stopTime = stopTimeList.get(i);
         final boolean hasDeparture = stopTime.hasDepartureTime();
         final boolean hasArrival = stopTime.hasArrivalTime();
+        if (isTimepoint(stopTime)) {
```

Comment:
This validation rule does not need the whole stop_times.txt table. Please make it a SingleEntityValidator instead.

Reply:
Thanks! Modified in https://github.com/MobilityData/gtfs-validator/pull/798/commits/f44acbb99a2cdcba38daf116e41034ad127b6a7a.

Reply:
I was wondering about this. Logic for `StopTimeWithDepartureBeforeArrivalTimeNotice` could potentially be moved to the same SingleEntityValidator as well, right?

Looks like there are some other FileValidators that could be moved to SingleEntityValidators, including:
* CalendarServiceDateValidator
* FeedServiceDateValidator

Reply:
https://github.com/MobilityData/gtfs-validator/issues/804 has been opened to keep track of this.